### PR TITLE
Two test cleanups

### PR DESCRIPTION
**Pull request by jayofdoom**
(two commits)
1) Move test data into text files that are loaded
2) Get rid of unneeded DDOEMCLI_FORMAT_OUT output being added in the list devices test.

### CODE REVIEW

File: onmetal_ironic_hardware_manager/tests/manager.py [hunk 1]
```diff
@@ -104,8 +69,7 @@ def _mock_file(self, mocked_open, contents):
     @mock.patch.object(utils, 'execute')
     def test__list_lsi_devices(self, mocked_execute):
         mocked_execute.side_effect = [
-            (DDOEMCLI_LISTALL_OUT, ''),
-            (DDOEMCLI_FORMAT_OUT, ''),
+            (DDOEMCLI_LISTALL_OUT, '')
```
Comment (jimrollenhagen) [inline onmetal_ironic_hardware_manager/tests/manager.py]: Why remove DDOEMCLI_FORMAT_OUT here? Does this still pass tests, and why?

File: onmetal_ironic_hardware_manager/tests/manager.py [hunk 2]
```diff
@@ -28,49 +28,14 @@
...
-    '\n'
-    'Seagate WarpDrive Management Utility: Execution completed successfully.\n'
-)
+
+def _read_file(test_data):
+    filename = os.path.join(os.path.dirname(__file__), test_data)
+    with open(filename, 'r') as data:
+        return data.read()
+
+DDOEMCLI_FORMAT_OUT = _read_file('data/ddoemcli_format_out.txt')
```
Comment (aweeks) [inline onmetal_ironic_hardware_manager/tests/manager.py]: DDOEMCLI_FORMAT_OUT is unused?

File: onmetal_ironic_hardware_manager/tests/manager.py (hunk 1 above)
Comment (jimrollenhagen) [inline onmetal_ironic_hardware_manager/tests/manager.py]: Oh, I see it. execute is only called once.

File: onmetal_ironic_hardware_manager/tests/manager.py (hunk 2 above)
Comment (jimrollenhagen) [inline onmetal_ironic_hardware_manager/tests/manager.py]: Still used in another function https://github.com/rackerlabs/onmetal-ironic-hardware-manager/blob/master/onmetal_ironic_hardware_manager/tests/manager.py#L125

**Comment by jimrollenhagen** [comment]
LGTM :+1:

**Comment by aweeks** [comment]
If there is a good reason for including DDOEMCLI_FORMAT_OUT, but not (apparently) using it, then +1

**Comment by pcsforeducation** [comment]
+1

**Comment by aweeks** [comment]
Ignore my previous comment, github search is bad.  +1

**Pull request merged**